src/main.js

```diff
@@ -45,13 +46,20 @@ async function main() {
     logger.info('Import and enrichment done');
   });
 
+  dumpAndPushS3Queue.process(async function() {
+    logger.info('dumpAndPushS3Queue.run - Started');
+    await steps.dumpAndPushS3(configuration);
+    logger.info('dumpAndPushS3Queue.run - Ended');
+    notificationQueue.add({}, { ...jobOptions, attempts: 1 });
```

Comment:
Comme le S3 est en dehors du réseau, je propose de configurer le nombre de tentatives via une variable d'environnement
```suggestion
    notificationQueue.add({}, { ...jobOptions, attempts });
```